docs/deployment/config_descriptions.md

```diff
@@ -211,6 +212,44 @@ Configuration for the Amazon Elasticsearch Service (ES) instance.  You can updat
           comparison_operator: GreaterThanThreshold
           threshold: '{{es.instanceCount}}'
           metric: Nodes
+```
+
+## sns
+Cumulus supports configuration and deployment of SNS topics and subscribers using `app/config.yml`. In the following code snippets we'll see an example topic and subscriber configuration.
+
+```yaml
+sns:
+  # this topic receives all the updates from
+  # step functions
+  sftracker:
+    subscriptions:
+      lambda:
+        endpoint:
+          function: Fn::GetAtt
+          array:
+            - sns2elasticsearchLambdaFunction
+            - Arn
+        protocol: lambda
+```
+
+The above code is an example of configuration for an SNS topic that will be called `sftrackerSns` in the resulting `cloudformation.yml` file. More information for each of the individual attributes can be found in [AWS SNS Topic Documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sns-topic.html).
```

Comment:
I think a short sentence explaining what the config does might be helpful. 

> This configuration creates an SNS topic named `<stackname>-sftracker` and subscribes the resource named `sns2elasticsearchLambdaFunction` to that topic so that it will be triggered when any messages are added to that topic.

Opinion: I think it might be preferable to explain the outcome for users in terms of named resources than what the cloudformation.yml says.

Reply:
https://github.com/nasa/cumulus/pull/943/commits/191a785964971a1312c0cae59d681abcecccc645